lib/oxidized/input/tftp.rb

```diff
@@ -28,14 +28,10 @@ def cmd file
       config.rewind
       config.read
     end
-    
-    private
-    
-    def disconnect
+  
+    private def disconnect
       # TFTP uses UDP, there is no connection to close
-    ensure
```

Comment:
This is presumably restructured to prevent Rubocop from breaking, but might be better as a separate commit / issue (can't immediately see why the `ensure` clause was here in the first place). Perhaps something along these lines to make this a pure Rubocop implementation commit in .rubocop.yaml instead?

```yaml
AllCops:
  Exclude:
    - 'lib/oxidized/input/tftp.rb'
```